src/App.js

```diff
@@ -28,7 +32,30 @@ function App() {
               </RequireAuth>
             }
           />
-          <Route path='/video/:id' element={<SingleVideoPage />} />
+          <Route
+            path='/watchlater'
+            element={
+              <RequireAuth>
+                <WatchLaterPage />
+              </RequireAuth>
+            }
+          />
+          <Route
+            path='/liked'
+            element={
+              <RequireAuth>
+                <LikedVideosPage />
+              </RequireAuth>
+            }
+          />
+          <Route
+            path='/history'
+            element={
+              <RequireAuth>
+                <HistoryPage />
+              </RequireAuth>
+            }
+          />
```

Comment:
Instead of wrapping requires auth everytime you can do something like [this](https://github.com/sudipkundu999/CobraTube/blob/devSudip/src/App.js#L40)

Reply:
Done thanks for pointing it out